### ISSUE TITLE
Country PDF Text too Small to Read on iPhone

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#47]
While looking at the application to check on the PDFs that were added to the "Sweat and Toil" country screens, I noticed that the text for the PDFs is so small to read even in landscape mode. Having the PDF available on an iPhone only makes it difficult for people to read without zooming in and having to scroll left and right to read the content.

**Comment by jfc3-dol** [comment]
Split this issue into three issues so we can track the small text and VoiceOver are not reading the heading issue, which is an issue is #48 - "VoiceOver Not Reading Country PDF Headings Correctly" and the "VoiceOver Not Reading Country PDF Tables Correctly" is issue #49.

**Comment by carolynhuang** [comment]
Cannot be resolved given limits in technology. As discussed, PDFs are not something that can be fixed now because of software limitations on mobile or tablet devices to read them correctly.

**Comment by jfc3-dol** [comment]
Country pages do have a link to the countries web page which is more accessible than the PDF and an alternative to the PDF. 

Closing issue.
